bakerytest/bakerytest.go

```diff
@@ -276,7 +276,7 @@ func (d *InteractiveDischarger) wait(w http.ResponseWriter, r *http.Request) {
 // particular interaction is complete. It causes any waiting requests to
 // return. If err is not nil then it will be returned by the
 // corresponding /wait request.
-func (d *InteractiveDischarger) FinishInteraction(w http.ResponseWriter, r *http.Request, err error) {
+func (d *InteractiveDischarger) FinishInteraction(w http.ResponseWriter, r *http.Request, cavs []checkers.Caveat, err error) {
```

Comment:
It's a bit of a pity that this is a backwardly incompatible change, but I guess it's not too bad because it's in test code not production code.

If we cared about backward compatibility I guess we could keep the old
method and add FinishInteractionWithCaveats, but as you said online,
the API is very new so probably no-one is using it.